classes/tank.class.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+class Tank{
+    public $life;
+    public $attack;
+    public $defence;
+    public $moves;
+    // TODO: id tank (variabila static in clasa abstracta?)
+
+    protected bool $isDefending = true;
+    protected $armorBonus = 20;
```

Comment:
I think you can make a method from this property .What do you think ?

Reply:
I agree @RobertPlaiasu but is not an issue.  use KISS. We could add more things later.